config/clusters/leap/common.values.yaml

```diff
@@ -150,7 +152,7 @@ basehub:
             mem_guarantee: 52G
             node_selector:
               node.kubernetes.io/instance-type: n1-standard-16
-
+          profile_options: *profile_options
```

Comment:
On mobile, but this looks like a yaml error putting options outside a profile list element

Reply:
On computer now, this seems correct! I got tricked by an inconsistency on placement in relation to other configuration.